www_src/components/option-panel/option-panel.jsx

```diff
@@ -0,0 +1,84 @@
+var React = require('react/addons');
+var classNames = require('classnames');
+
+// Internal -- just the container
+var OptionPanel = React.createClass({
+  render: function () {
+    return (<div className="option-panel">
+      {this.props.children}
+    </div>);
+  }
+});
+
+var Checkbox = React.createClass({
+  mixins: [React.addons.LinkedStateMixin],
+  getDefaultProps: function () {
+    return {
+      linkState: this.linkState,
+      checkedValue: true,
+      uncheckedValue: false
+    };
+  },
+  onChange: function (e) {
+    var val = e.target.checked ? this.props.checkedValue : this.props.uncheckedValue;
```

Comment:
`this.state. ...` rather than `this.props. ...`, since this value changes throughout the component's life cycle?